src/api/json/catalog.json

```diff
@@ -5463,6 +5463,12 @@
       "fileMatch": ["**/sites/*/config.yaml", "**/sites/*/config.yml"],
       "url": "https://json.schemastore.org/typo3.json"
     },
+    {
+      "name": "typos.toml",
+      "description": "typos configuration file",
```

Comment:
Intentionally lowercase here too, for same reasoning as in https://github.com/SchemaStore/schemastore/pull/4587#discussion_r2006359841